code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -171,6 +171,12 @@ There are several things that need to be remembered:
 				else
 					handled_by_bodytype = FALSE
 
+			if(dna.species.bodytype & BODYTYPE_VOX)
+				if(U.supports_variations_flags & CLOTHING_VOX_VARIATION)
+					icon_file = U.worn_icon_vox || VOX_UNIFORM_FILE
+				else
+					handled_by_bodytype = FALSE
```

Comment:
I'm 99% sure this is redundant, because passing `null` into `icon_exists()` is a valid command and will always return false.

You're just copying tesh code though so I don't blame you. I'll give it a look over myself in the future, not a big deal